### PR TITLE
lost pools

### PR DESCRIPTION
**Pull request by AbigailTCameron**

### CODE REVIEW

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            _timelock
+        )
+    {}
+
+    /// @notice **** State Mutations ****
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        IMiniChef(miniChef).harvest(poolId, address(this));
+
+        /// @dev Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)
```
Comment (zeroecksYeti) [inline contracts/strategies/pangolin/strategy-png-avax-lost.sol:35]: I don't know if NatSpec tags are supposed to be used as regular comments. I think they're meant to be headers for things like functions and interfaces. I would ask @Auroter for guidance on this

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        if (_avax > 0) {                                    
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+
+        /// @dev Check token balances, take fee for each token, then update balances
+        uint256 _lost = IERC20(lost).balanceOf(address(this));
+        uint256 _png = IERC20(png).balanceOf(address(this));
+
+        if (_lost > 0) {
+            _swapPangolin(lost, wavax, _lost);
```
Comment (zeroecksYeti) [inline contracts/strategies/pangolin/strategy-png-avax-lost.sol:46]: I don't see the usual `safeApprove()` lines here, do we not need them anymore?

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            uint256 _keep = _wavax.mul(keep).div(keepMax);
+            if (_keep > 0) {
+                _takeFeeWavaxToSnob(_keep);
+            }
+            _wavax = IERC20(wavax).balanceOf(address(this));
+            IERC20(wavax).safeApprove(pangolinRouter, 0);
+            IERC20(wavax).safeApprove(pangolinRouter, _wavax.div(2));   
+            _swapPangolin(wavax, lost, _wavax.div(2)); 
+        }      
+
```
Comment (zeroecksYeti) [inline contracts/strategies/pangolin/strategy-png-avax-lost.sol:65]: Remove extra line

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol (hunk 2 above)
Comment (AbigailTCameron) [reply contracts/strategies/pangolin/strategy-png-avax-lost.sol:46]: they are already inside the _swapPangolin function (at the strategy base)

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol (hunk 1 above)
Comment (AbigailTCameron) [reply contracts/strategies/pangolin/strategy-png-avax-lost.sol:35]: I didn't add that comment, but I could remove it

File: contracts/strategies/traderJoe/strategy-joe-avax-lost.sol [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        
+        // Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)
+        uint256 _avax = address(this).balance;
+        if (_avax > 0) {                                    
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+        
+        // Check token balances, swap token into WAVAX
+        uint256 _lost = IERC20(lost).balanceOf(address(this));       
+        // Swap APE into WAVAX
```
Comment (zeroecksYeti) [inline contracts/strategies/traderJoe/strategy-joe-avax-lost.sol:44]: // Swap LOST into WAVAX

**Review by zeroecksYeti — COMMENTED** [review]
A few comments, mostly on the PNG strat

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol (hunk 2 above)
Comment (zeroecksYeti) [reply contracts/strategies/pangolin/strategy-png-avax-lost.sol:46]: Copy

File: contracts/strategies/pangolin/strategy-png-avax-lost.sol (hunk 1 above)
Comment (zeroecksYeti) [reply contracts/strategies/pangolin/strategy-png-avax-lost.sol:35]: I think if you change them to regular comments it will be fine

**Review by zeroecksYeti — APPROVED** [review]
LGTM. Deployed and ready to merge

**Pull request merged**